ansible/roles/geoblacklight/tasks/geoblacklight_setup.yml

```diff
@@ -6,7 +6,6 @@
     owner: "{{ project_user }}"
     group: "{{ project_group }}"
     state: directory
-    recurse: yes
```

Comment:
We don't want to recurse because that means the files that must be owned by the project_run_user will be changed to the project_user and back right? But if we aren't recursing, this only handles the directory. If we don't have problems with the rest of the application, should this task be removed in another ticket?